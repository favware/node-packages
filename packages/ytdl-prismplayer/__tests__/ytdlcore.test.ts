import {
  chooseFormat,
  downloadFromInfo,
  filterFormats,
  getBasicInfo,
  getInfo,
  getURLVideoID,
  getVideoID,
  validateID,
  validateURL
} from '../src';

describe('ytdl-core validation', () => {
  test('should export chooseFormat', () => {
    expect(chooseFormat).toBeDefined();
    expect(chooseFormat).toMatchSnapshot();
    expect(chooseFormat).toBeInstanceOf(Function);
  });

  test('should export downloadFromInfo', () => {
    expect(downloadFromInfo).toBeDefined();
    expect(downloadFromInfo).toMatchSnapshot();
    expect(downloadFromInfo).toBeInstanceOf(Function);
  });

  test('should export filterFormats', () => {
    expect(filterFormats).toBeDefined();
    expect(filterFormats).toMatchSnapshot();
    expect(filterFormats).toBeInstanceOf(Function);
  });

  test('should export getBasicInfo', () => {
    expect(getBasicInfo).toBeDefined();
    expect(getBasicInfo).toMatchSnapshot();
    expect(getBasicInfo).toBeInstanceOf(Function);
  });

  test('should export getInfo', () => {
    expect(getInfo).toBeDefined();
    expect(getInfo).toMatchSnapshot();
    expect(getInfo).toBeInstanceOf(Function);
  });

  test('should export getURLVideoID', () => {
    expect(getURLVideoID).toBeDefined();
    expect(getURLVideoID).toMatchSnapshot();
    expect(getURLVideoID).toBeInstanceOf(Function);
  });

  test('should export getVideoID', () => {
    expect(getVideoID).toBeDefined();
    expect(getVideoID).toMatchSnapshot();
    expect(getVideoID).toBeInstanceOf(Function);
  });

  test('should export validateID', () => {
    expect(validateID).toBeDefined();
    expect(validateID).toMatchSnapshot();
    expect(validateID).toBeInstanceOf(Function);
  });

  test('should export validateURL', () => {
    expect(validateURL).toBeDefined();
    expect(validateURL).toMatchSnapshot();
    expect(validateURL).toBeInstanceOf(Function);
  });
});
