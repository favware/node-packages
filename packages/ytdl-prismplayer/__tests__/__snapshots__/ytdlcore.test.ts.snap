// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ytdl-core validation should export chooseFormat 1`] = `[Function]`;

exports[`ytdl-core validation should export downloadFromInfo 1`] = `[Function]`;

exports[`ytdl-core validation should export filterFormats 1`] = `[Function]`;

exports[`ytdl-core validation should export getBasicInfo 1`] = `[Function]`;

exports[`ytdl-core validation should export getInfo 1`] = `[Function]`;

exports[`ytdl-core validation should export getURLVideoID 1`] = `[Function]`;

exports[`ytdl-core validation should export getVideoID 1`] = `[Function]`;

exports[`ytdl-core validation should export validateID 1`] = `[Function]`;

exports[`ytdl-core validation should export validateURL 1`] = `[Function]`;
